src/PKSim.Core/Services/BuildingBlockParametersToSimulationUpdater.cs

```diff
@@ -65,9 +72,28 @@ public ICommand UpdateParametersFromBuildingBlockInSimulation(IPKSimBuildingBloc
          updateCommands.CommandType = PKSimConstants.Command.CommandTypeUpdate;
          updateCommands.Description = PKSimConstants.Command.UpdateBuildingBlockCommandDescription(buildingBlockType, templateBuildingBlock.Name, simulation.Name);
          _executionContext.UpdateBuildingBlockPropertiesInCommand(updateCommands, simulation);
+
+         synchronizeBuildingBlocks(templateBuildingBlock, simulation);
          return updateCommands;
       }
 
+      /// <summary>
+      ///    We need to make sure that once the simulation has been updated with the building block, depending building blocks
+      ///    are also updated
+      ///    For instance, if we update the individual in the simulation, we will also update all expression profile (since
+      ///    expression profile are linked to the individual)
+      /// </summary>
+      private void synchronizeBuildingBlocks(IPKSimBuildingBlock templateBuildingBlock, Simulation simulation)
```

Comment:
BB => simulation. We need to make sure that the expression profile in the sim matches the template AFTER THE UPDATE